test/features/sdk_flow.feature

```diff
@@ -113,3 +113,32 @@ Feature: SDK File Upload Tests
       | type |
       |      |
       | pdf  |
+
+  Scenario Outline: I can navigate to the previous screen
```

Comment:
Then finally I'd rename it to something like "I can navigate to the second last step of the flow and then go back to the beginning"